certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json

```diff
@@ -0,0 +1,800 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zoho ZeptoMail",
+    "description": "ZeptoMail is a transactional email sending service from Zoho. You can directly send these automated emails from your application or website.",
+    "version": "1.0",
+    "contact": {
+      "name": "Zoho ZeptoMail",
+      "email": "zmintegration@zohomail.com",
+      "url": "https://www.zoho.com/zeptomail/contact.html?src=pd-menu"
+    }
+  },
+  "host": "zeptomail.zoho.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/portal/v1.0/mailagents": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "mailagent_name": {
+                        "type": "string",
+                        "description": "Mail agent name"
+                      },
+                      "created_time": {
+                        "type": "string",
+                        "description": "Mail agent created time"
+                      },
+                      "mailagent_id": {
+                        "type": "string",
+                        "description": "Mail agent id"
+                      },
+                      "domain": {
+                        "type": "string",
+                        "description": "Domain"
+                      },
+                      "domain-verification-status": {
+                        "type": "string",
+                        "description": "Domain verification status"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Mail agent description"
+                      },
+                      "user_connection_status": {
+                        "type": "string",
+                        "description": "User connection status"
+                      },
+                      "mailagent_key": {
+                        "type": "string",
+                        "description": "Mail agent key"
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "status"
+                      }
+                    }
+                  },
+                  "description": "data",
+                  "minLength": 1
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get mail agent",
+        "description": "Get mail agent",
```

Comment:
@zohomail all operation descriptions should be different that the summary. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operations' descriptions in the swagger file.